### ISSUE TITLE
worldwind java 2.1.0 goes wrong on ubuntu 20.04

### ISSUE THREAD

**Issue by youngcle** [NASAWorldWind/WorldWindJava#197]
Description

worldwind java 2.1.0 goes wrong on ubuntu 20.04
in the previous version, like ubuntu 18.04 ,Centos 7 linux distribution, worldwind java works well.
i think there is something about **opengl** goes wrong.

i hope the official developer can see this,and fix it

Expected behavior:
the 3D globe window cannot show up.
and errors showed in a dialogue window.

Actual behavior:
Steps to Reproduce

    put the program in ubuntu 20.04
    run the application,then goes wrong

Reproduces how often:
everytime
Operating System and Version

ubuntu 20.04
Additional Information

glxinfo in my ubuntu 20.04

name of display: :0
display: :0 screen: 0
direct rendering: Yes
server glx vendor string: SGI
server glx version string: 1.4
server glx extensions:
GLX_ARB_create_context, GLX_ARB_create_context_no_error,
\
GLX_SGIX_visual_select_group, GLX_SGI_make_current_read,
GLX_SGI_swap_control
client glx vendor string: Mesa Project and SGI
client glx version string: 1.4
client glx extensions:
GLX_ARB_context_flush_control, GLX_ARB_create_context,

GLX version: 1.4
GLX extensions:
GLX_ARB_create_context, GLX_ARB_create_context_no_error,

Extended renderer info (GLX_MESA_query_renderer):
Vendor: Intel (0x8086)
Device: Mesa Intel(R) HD Graphics 520 (SKL GT2) (0x1916)
Version: 20.0.4
Accelerated: yes
Video memory: 3072MB
Unified memory: yes
Preferred profile: core (0x1)
Max core profile version: 4.6
Max compat profile version: 4.6
Max GLES1 profile version: 1.1
Max GLES[23] profile version: 3.2
OpenGL vendor string: Intel
OpenGL renderer string: Mesa Intel(R) HD Graphics 520 (SKL GT2)
OpenGL core profile version string: 4.6 (Core Profile) Mesa 20.0.4
OpenGL core profile shading language version string: 4.60
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile
OpenGL core profile extensions:
GL_3DFX_texture_compression_FXT1, GL_AMD_conservative_depth,

OpenGL version string: 4.6 (Compatibility Profile) Mesa 20.0.4
OpenGL shading language version string: 4.60
OpenGL context flags: (none)
OpenGL profile mask: compatibility profile
OpenGL extensions:
GL_3DFX_texture_compression_FXT1, GL_AMD_conservative_depth,

OpenGL ES profile version string: OpenGL ES 3.2 Mesa 20.0.4
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
OpenGL ES profile extensions:
GL_AMD_performance_monitor, GL_ANDROID_extension_pack_es31a,

**Comment by Beak-man** [comment]
@youngcle Have you tried the latest commit in the `develop` branch? We just finished a round of testing under Ubuntu 20.04.

**Comment by FJBDev** [comment]
> 
> 
> @youngcle Have you tried the latest commit in the `develop` branch? We just finished a round of testing under Ubuntu 20.04.

Can you please explain how to use _"the latest commit in the develop branch? "_

**Comment by youngcle** [comment]
> @youngcle Have you tried the latest commit in the `develop` branch? We just finished a round of testing under Ubuntu 20.04.

First of all,thanks for the reply.
I've cloned the develop branch of the worldwindjava.

I found that the offical team changed the JDK version to 10 or above,

if it compiled with JDK 8，some code would goes wrong.

anyway, we can get it work in ubuntu 20.04 again.


I'm very glad that the offical team is still working on this great project.

Bravo!
